packages/client/src/clients/in-app/message-client.ts

```diff
@@ -0,0 +1,98 @@
+import Knock from "../../knock";
+import { NetworkStatus, isRequestInFlight } from "../../networkStatus";
+
+import { InAppChannelClient } from "./channel-client";
+import {
+  FetchInAppMessagesOptions,
+  InAppMessageClientOptions,
+  InAppMessagesQueryInfo,
+} from "./types";
+
+const defaultOptions: Pick<InAppMessageClientOptions, "archived"> = {
+  archived: "exclude",
+};
```

Comment:
Fwiw, by default we already set to "exclude" in the backend so not sure if this is strictly necessary.

Reply:
Sounds good! Was following the existing `feed` but we can simplify this